public_html/wp-content/mu-plugins/theme-templates/bootstrap.php

```diff
@@ -144,8 +198,10 @@ function inject_offline_template( $template_path ) {
  * @return array|false
  */
 function add_offline_template_cachebuster( $entry ) {
+	$page = get_offline_content();
+
 	if ( $entry && isset( $entry['revision'] ) ) {
-		$entry['revision'] .= ';' . filemtime( __DIR__ . '/templates/offline.php' );
+		$entry['revision'] .= ';' . md5( $page['content'] );
```

Comment:
Perhaps include both?

```suggestion
		$entry['revision'] .= ';' . filemtime( __DIR__ . '/templates/offline.php' ) . md5( $page['content'] );
```

This would ensure the revision is bumped whenever the content changes _or_ the template is updated.